srcpkgs/cryptomator-cli/template

```diff
@@ -0,0 +1,36 @@
+# Template file for 'cryptomator-cli'
+pkgname=cryptomator-cli
+version=0.4.0
+revision=1
+wrksrc="cli-${version}"
+makedepends="apache-maven"
+depends="openjdk11 davfs2"
+short_desc="Cryptomator Command Line Interface "
+maintainer="nielznl <nielznl@protonmail.com>"
+license="GPL-3.0-only"
+homepage="https://cryptomator.org"
+distfiles="https://github.com/cryptomator/cli/archive/${version}.tar.gz"
+checksum=53a52dab25ca9d0e565e2066050fbe3933ffacb7ce14eb2f965ca818e8e4a49b
+
+do_build() {
+	mvn clean install
+}
+
+post_build() {
+	# Make the jar file linux executable
+	cd target
+	echo '#!/bin/sh
+	MYSELF=$(which "$0" 2>/dev/null)
+	[ $? -gt 0 -a -f "$0" ] && MYSELF="./$0"
+	java=java
+	if test -n "$JAVA_HOME"; then
+	    java="$JAVA_HOME/bin/java"
+	fi
+	exec "$java" $java_args -jar $MYSELF "$@"
+	exit 1' > stub.sh
+	cat stub.sh "${pkgname}-${version}.jar" > ${pkgname}
```

Comment:
Assuming I understood this correctly, you shouldn't do this, the preferred solution is to make a wrapper script in `files/` and `exec java -jar` in it.

Also, `which` isn't installed by default, you can just use `command -v`

Reply:
Yeah I was thinking about something like that. I created the wrapper script to be put in /bin and I am now copying the jar file to /usr/share/${pkgname}.

Reply:
I thought /usr/share/${pkgname} would be the right location